.github/workflows/release.yml

```
@@ -0,0 +1,99 @@
+name: Release
+on:
+  push:
+    branches:
+      - main
+      - '20.[0-9]+.[0-9]+'
+jobs:
+  release:
+    if: "!contains(github.event.head_commit.message, '[skip release]')"
+    name: Release
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    steps:
+      - uses: Brightspace/third-party-actions@actions/checkout
+        with:
+          persist-credentials: false
+      - uses: Brightspace/third-party-actions@actions/setup-node
+        with:
+          node-version: '14'
+          cache: 'npm'
+      - name: Install dependencies
+        run: |
+          npm ci
+          npm install frau-publisher --no-save
+      - name: Build
+        run: |
+          npm run build
+          npm run build:lang
+        env:
+          NODE_OPTIONS: '--max-old-space-size=6656'
+      - name: Get Branch Name
+        id: get-branch
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+        shell: bash
+      - name: Get Active LMS Release
+        id: get-lms-version
+        uses: Brightspace/lms-version-actions/get-lms-release@main
+        with:
+          RALLY_API_KEY: ${{ secrets.RALLY_API_KEY }}
+      - name: Get Release
+        id: get-release
+        run: |
+          if [ "${{ steps.get-branch.outputs.branch }}" == "main" ]; then
+            echo "::set-output name=release::$(echo ${{ steps.get-lms-version.outputs.LMS_VERSION }})"
+          else
+            echo "::set-output name=release::$(echo ${{ steps.get-branch.outputs.branch }})"
+          fi
+        shell: bash
```

Prefer passing through environment variables rather than substituting directly into the script to avoid injection possibilities.

```suggestion
      - name: Get Release
        id: get-release
        run: |
          if [ "${BRANCH}" == "main" ]; then
            echo "::set-output name=release::${LMS_VERSION}"
          else
            echo "::set-output name=release::${BRANCH}"
          fi
        shell: bash
        env:
          BRANCH: ${{ steps.get-branch.outputs.branch }}
          LMS_VERSION: ${{ steps.get-lms-version.outputs.LMS_VERSION }}
```